include/taskolib/taskolib.h

```diff
@@ -22,17 +22,17 @@
 
 // SPDX-License-Identifier: LGPL-2.1-or-later
 
-#ifndef TASKOMAT_TASKOMAT_H_
-#define TASKOMAT_TASKOMAT_H_
+#ifndef TASKOLIB_TASKOLIB_H_
+#define TASKOLIB_TASKOLIB_H_
 
-#include "taskomat/Context.h"
-#include "taskomat/exceptions.h"
-#include "taskomat/Executor.h"
-#include "taskomat/Sequence.h"
-#include "taskomat/Step.h"
-#include "taskomat/Taskomat.h"
-#include "taskomat/time_types.h"
-#include "taskomat/VariableName.h"
+#include "taskolib/Context.h"
+#include "taskolib/exceptions.h"
+#include "taskolib/Executor.h"
+#include "taskolib/Sequence.h"
+#include "taskolib/SequenceManager.h"
```

Comment:
Arrrgh ... I forget to add `SequenceManager.h` to `taskolib.h`. Sorry!

Reply:
Happens.... thus my question if we want to automate a test for the headers (like we have in GUL).

Reply:
Yes, sure! I would very much appreciate a test for this case also to avoid some inconsistency for the users.